src/main/java/de/dataelementhub/model/handler/export/ExportHandler.java

```diff
@@ -43,7 +43,7 @@ public static void export(
     new File(destination).mkdir();
     nonExportable.clear();
     try {
-      Export export = new Export();
+      DataTransfer export = new DataTransfer();
```

Comment:
Same as above...I think we should come up with a better name

Reply:
Renamed to ImportExport